doc/changelog.md

```diff
@@ -31,6 +32,8 @@ Description
 
 Detailed Notes
 
+- Added type checking to params on model
```

Comment:
```suggestion
The serializer would fail if a parameter for a Model or Ensemble was specified as a numpy dtype. The constructors for these methods now validate that the input is number-like and convert them to strings
```

Reply:
changed